R/combinations-isobolograms.R

```diff
@@ -338,11 +338,11 @@ calculate_Loewe <- function(
 }
 
 
-get_isocutoffs <- function(df_mean, normalization_type) {
-  if (min(df_mean[[normalization_type]], na.rm = TRUE) > 0.7) {
+get_isocutoffs <- function(df_mean, normalization_type, data_col = "average") {
```

Comment:
Is there any reason why all the functions in `combinations-isobologram.R` have no tests?

Reply:
Indeed, there are no tests. I will try to create some even basic tests for this function